tools/cli/commands/changelog.js

```diff
@@ -290,81 +289,92 @@ async function checkSpecialProjects( needChangelog ) {
  * @param {argv} argv - the arguments passed.
  */
 async function changelogAdd( argv ) {
-	if ( argv._[ 1 ] === 'add' && ! argv.project ) {
-		const needChangelog = await checkChangelogFiles();
-		const uniqueProjects = await checkSpecialProjects( needChangelog );
+	let needChangelog;
+	const defaultProjects = [];
+	const uniqueProjects = [];
+	const defaultTypes = {
+		security: 'Improves or modifies the security of the project.',
+		added: 'Added new functionality.',
+		changed: 'Changed existing functionality.',
+		deprecated: 'Deprecated existing functionality.',
+		removed: 'Removed existing functionality.',
+		fixed: 'Fixed a bug.',
+	};
+
+	if ( argv.project ) {
+		needChangelog = [ argv.project ];
+	}
 
-		// If we don't detect any modified projects, shortcircuit to default changelogger.
-		if ( needChangelog.length === 0 && uniqueProjects.length === 0 ) {
+	// If we weren't passed a project, check if any projects need changelogs.
+	if ( argv._[ 1 ] === 'add' && ! argv.project ) {
+		needChangelog = await checkChangelogFiles();
+		if ( needChangelog.length === 0 ) {
 			console.log(
 				chalk.green(
-					'Did not detect a touched project that still need a changelog. You can still add a changelog manually.'
+					'Did not detect a touched project that still needs a changelog. You can still add a changelog manually by running `jetpack changelog add <type/project>'
 				)
 			);
-			changelogArgs( argv );
 			return;
 		}
+	}
 
-		const promptType = await changelogAddPrompt( argv, needChangelog, uniqueProjects );
+	const projectChangeTypes = await getProjectChangeTypes( needChangelog );
 
-		// Bail if user doesn't want to auto-add.
-		if ( ! promptType.autoAdd && ! promptType.autoPrompt ) {
-			console.log(
-				chalk.green(
-					`Auto changelog cancelled. You can run 'jetpack changelog add [project-type/project]' to add changelogs individually.`
-				)
-			);
-			return;
+	// Iterate over the project types, and if it's not default, add it to uniqueProjects.
+	for ( const proj in projectChangeTypes ) {
+		if ( projectChangeTypes[ proj ] !== 'default' ) {
+			uniqueProjects.push( proj );
+		} else {
+			defaultProjects.push( proj );
+			projectChangeTypes[ proj ] = defaultTypes;
 		}
+	}
 
-		// Auto add the changelog files for the projects that we can:
-		if ( promptType.autoAdd ) {
-			console.log(
-				chalk.green( `Running auto changelogger for ${ needChangelog.length } project(s)!` )
-			);
-			const response = await promptChangelog( argv, needChangelog );
-			for ( const proj of needChangelog ) {
-				argv = await formatAutoArgs( proj, argv, response );
-				await changelogArgs( argv );
-			}
+	// Confirm what projects we're adding a changelog to, and how we want to add them.
+	const promptConfirm = argv.project
+		? true
```

Comment:
If we do this, the behavior is a little more consistent between e.g. `jetpack changelog add plugins/beta` and `jetpack changelog add plugins/jetpack`.
```suggestion
		? { separateChangelogFiles: true }
```